servicecontrol/transports.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Transport support
+reviewed: 2021-09-01
+---
+ServiceControl can be configured to use one of the supported [transports](/transports/) listed below using the ServiceControl Management application:
+
+* [Azure Service Bus](/transports/azure-service-bus)
+* [Azure Service Bus - Endpoint-oriented topology](/transports/azure-service-bus/legacy/topologies.md#versions-7-and-above-endpoint-oriented-topology)
+* [Azure Service Bus - Forwarding topology](/transports/azure-service-bus/legacy/topologies.md#versions-7-and-above-forwarding-topology)
+* [Azure Storage Queues](/transports/azure-storage-queues/)
+* [Amazon Simple Queue Service (SQS)](/transports/sqs/)
+* [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
+* [RabbitMQ - Conventional routing topology](/transports/rabbitmq/routing-topology.md#conventional-routing-topology)
+* [RabbitMQ - Direct routing topology](/transports/rabbitmq/routing-topology.md#direct-routing-topology)
+* [SQL Server](/transports/sql/)
+
+### Transport-specific features
+
+#### Transport adapters
+
+Certain transport features are not supported natively by ServiceControl and will require a [transport adapter](/servicecontrol/transport-adapter). Contact support@particular.net for further guidance.
```

Comment:
Why do we refer to support here? Is this a gap in our doco? Should we raise an issue for this?

Reply:
This is copied text that I did not write myself. Yeah, maybe raise an issue.